docs/apm/traces/working-with-tracing-data/services-list-map.md

```diff
@@ -0,0 +1,119 @@
+  * Arrows map out connection activity between services and their presence from the last 72 hours.
+  * Circle and arrows with dotted lines represent services or connections that have been inactive an hour or more.
+* **Size** of services: Size is based on service activity, where large circles are more active compared to smaller circles that are less active.  
+
+
+## Anomaly Detection
+
+An anomaly is a spike in latency and errors or a dip in requests. In both the **Services Map** and **Services List** views, automatic anomaly detection is enabled by default. 
+
+### Automatic
+
+With the **Worst Case** option selected (default), a service is displayed in a red circle if at least one of the KPIs (latency, errors, requests) has had an anomaly in the last 15 minutes AND the anomaly was not present 7 days ago at the same time. You can explicitly choose one of the other KPIs instead.
+
+<img src={useBaseUrl('img/traces/services-list-map-auto-detection-settings.png')} alt="auto service anomaly settings" width="400" />
+
+The standard [metric outlier operator](/docs/metrics/metric-charts/metrics-outliers) is used to detect this. You can adjust the sensitivity and **Learning window** of the outlier detection in the settings to make the logic more or less dependent on occasional spikes. 
+
+### Manual
+Manual anomaly detection allows you to specify different detection settings for each KPI. Click the toggle switch to manually enable or disable each KPI highlight as desired.
+
+<img src={useBaseUrl('img/traces/services-list-map-manual-detection-settings.png')} alt="services manual anomaly detection settings" width="400"/>
+
+
+
+
+## Services Dashboard Panels
+
+The **Services Map** and **Services List** panels, available out of the box, enable you to explore your application environment and review all traces from your Dashboard. You can duplicate or add multiple panels with different filtering or queries to refine views and support your organization.
+
+To add a Services panel to your Dashboard:
+
+1. Open an existing Dashboard or create a new Dashboard (**+New** > **Dashboard New**).
+1. On an existing Dashboard, click **Add Panel** > **Services**, or, on a new Dashboard, click **Services**.<br/> ![add-servicemap.png](/img/traces/add-servicemap.png)  
+1. When the panel configuration page opens, go to the **Visual Settings** > **Chart Type**, dropdown and select **Table** (Services List) or **Graph** (Services Map). <br/> ![create-servicemap.png](/img/traces/create-servicemap.png)
+1. Select from the dropdown menus to filter by the following:<br/>  ![filter-servicemap.png](/img/traces/filter-servicemap.png)
+   * **Application** (Optional). You can use this if your tracing data has the tag called `application=[app-name]`
+   * **Environment**. Your production, staging, or development environment name
```

Comment:
that's optional as well

Reply:
Application. Your application name. You can use this if your tracing data has the tag called application=[app-name] To pass the variables from dashboard filters, set application={{application}} 
Environment. Your production, staging, or development environment name. You can use this if your tracing data has the tag called deployment.environment=[environment-name] To pass the variables from dashboard filters, set deployment.environment={{deployment.environment}} 
Service(s). Your application name, which should be always there. To pass the variables from dashboard filters, set service={{service}}